CONTRIBUTING.md

```diff
@@ -1,12 +1,78 @@
-## Before you submit a new issue
-
-* Check if there's a simple solution in the
-  [Troubleshooting](https://github.com/npm/npm/wiki/Troubleshooting)
-  wiki.
-* [Search for similar
-  issues](https://github.com/npm/npm/search?q=Similar%20issues&type=Issues).
-* Ensure your new issue conforms to the [Contributing
-  Guidelines](https://github.com/npm/npm/wiki/Contributing-Guidelines).
-
-Participation in this open source project is subject to the [npm Code
-of Conduct](http://www.npmjs.com/policies/conduct).
+# npm CLI Contributor Guide
```

Comment:
I envisage this document getting longer. It might be nice to do a table of contents. As a community member (someone who tries to help out in the issue tracker?) I might see myself needing to scroll down to the bottom of the page a lot :D

So a table of contents where each line links to the relevant issue (this is possible with link IDs and such). Like was done in the troubleshooting guide ✨